src/NumField/NfAbs/Simplify.jl

```diff
@@ -345,7 +345,7 @@ function _find_prime(v::Vector{fmpz_poly})
       d1 = lcm(Int[x for (x, v) in FS])
       d = lcm(d1, d)
     end
-    if d < total_deg^2
+    if d <= total_deg^2
```

Comment:
Do you have an example where the code fails because of this? Or the result is suboptimal?